### PR TITLE
New view picker

### PR DESCRIPTION
**Pull request by charlesBochet**
And... VOILA!

https://github.com/twentyhq/twenty/assets/12035771/cf3fdf47-96e7-4be8-9b29-b9c6b9df6976

### CODE REVIEW

**Comment by mabdullahabaid** [comment]
Love the video, looks amazing!!

**Review by lucasbordeau — APPROVED** [review]
Maybe we could find a util for `view.key !== 'INDEX'`, appart from that nice work !

File: packages/twenty-front/src/modules/views/hooks/useHandleViews.ts [hunk 1]
```diff
@@ -63,45 +76,54 @@ export const useHandleViews = (viewBarComponentId?: string) => {
...
+      saveCurrentViewFilterAndSorts,
     ],
   );
 
   const changeViewInUrl = useCallback(
     (viewId: string) => {
-      setSearchParams((previousSearchParams) => {
-        previousSearchParams.set('view', viewId);
-        return previousSearchParams;
+      setSearchParams(() => {
```
Comment (Weiko) [inline packages/twenty-front/src/modules/views/hooks/useHandleViews.ts:112]: nit: I think you can directly use the constructor like this
```
setSearchParams(() => { return new URLSearchParams({ view: viewId }); });
```
or even
```
setSearchParams(new URLSearchParams({ view: viewId }));
```

**Review by Weiko — COMMENTED** [review]
Just a few comments, LGTM!

**Pull request merged**